ajax/libs/timecircles/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "timecircles",
+  "filename": "TimeCircles.min.js",
+  "version": "1.5.3",
+  "homepage": "http://git.wimbarelds.nl/TimeCircles/",
+  "authors": {
+    "name" : "Wim Barelds",
+    "email" : "wim.barelds@gmail.com",
+    "url" : "http://www.wimbarelds.nl/"
+  },
+  "description": "TimeCircles is a jQuery plugin that provides a nice looking way to either count down towards a certain time, or to count up from a certain time. The goal for TimeCircles is to provide a simple yet dynamic tool that makes it very easy to provide visitors an attractive countdown or timer.",
+  "keywords": [
+    "timer",
+    "stopwatch",
+    "count",
+    "countdown",
+    "time",
+    "date",
+    "datetime",
+    "clock",
+    "comingsoon",
+    "ui",
+    "construction"
+  ],
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/wimbarelds/TimeCircles"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/wimbarelds/TimeCircles.git",
+    "basePath": "inc/",
+    "files": [
+      "TimeCircles*.+(css|js)"
```

Comment:
current ext looks okay, once you add prefix in filemap, and currently all the files are needed, I think it's fine to ignore ext for now.